Documentation/security/restrict-pod-access.rst

```diff
@@ -0,0 +1,52 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _security_restrict_pod_access:
+
+========================================
+Restricting privileged Cilium pod access
+========================================
+
+This page shows you how to restrict priveleged access to Cilium pods by limiting access from the Kubernetes API, specifically from `kubernetes exec pod`_.
+
+.. include:: gsg_requirements.rst
+
+Background
+----------
+
+While the Cilium agent no longer requires a specific set of privileges and capabilities, the Cilium agent requires specific privileges to perform essential system and network operations. 
```

Comment:
While Cilium agent has made progress by reducing its set of privileges, it still requires some to perform essential system and network operations. ?

Reply:
Back story: Cilium used to have to be run with the `--privileged` flag but that is no longer true. I don't think the docs should be concerned with the history though. So I would suggest something like this: 

```The Cilium agent needs some specific Linux capabilities to perform essential system and network operations.```